crates/sui-tool/src/lib.rs

```diff
@@ -2,4 +2,484 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use anyhow::Result;
```

Comment:
I think this is fine but having the refactor in a separate PR can make this PR less intimidating to review.